vars/applier.groovy

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env groovy
+
+class ApplierInput implements Serializable{
+    String secretName = ""
+    String registryUrl = ""
+    String inventoryPath = ""
+    String requirementsPath = ""
+}
+
+def call(Map input) {
+    call(new ApplierInput(input))
+} 
+
+def call(ApplierInput input) {
+    script {
+        openshift.withCluster() {
+        def secretData = openshift.selector("secret/${input.secretName}").object().data
+        def encodedToken = secretData.token
+        env.TOKEN = sh(script:"set +x; echo ${encodedToken} | base64 --decode", returnStdout: true)
+        }
+
+        sh """
+            oc login ${input.registryUrl} --token=${env.TOKEN} --insecure-skip-tls-verify
+            ansible-galaxy install -r ${input.requirementsPath} --roles-path=galaxy
```

Comment:
A bit of a nitpick but let's be consistent with the options:
`ansible-galaxy install -r ${input.requirementsPath} -p galaxy` (personal preference) or
`ansible-galaxy install --role-file=${input.requirementsPath} --roles-path=galaxy`